.env

```diff
@@ -0,0 +1,19 @@
+NODE_ENV=development
```

Comment:
You need to add `.env` to your `~/.gitignore_global` :)

Reply:
oh woops, shouldn't it be in our `.gitignore` as well?